### PR TITLE
Luis learn more

### PR DESCRIPTION
**Pull request by LuisLizardo**

### CODE REVIEW

**Review by RochX — CHANGES_REQUESTED** [review]
I like the idea of the randomized carousel and the images used, but some courses that shouldn't be put on there are, such as SIP courses and Study Abroad "courses."

I also prefer @harshsepi0l's course details page over the the learn more page here.

@LuisLizardo you should talk to @harshsepi0l about this before the demo tomorrow.

This is just a visual change, so it won't effect my practice of the demo.

File: src/components/Carousel/Carousel.jsx [hunk 1]
```diff
@@ -1,107 +1,157 @@
...
         <div className="info-container">
-          <h1 className="carousel-top">
-            {carouselData[currentSlide].course_id +
-              ": " +
-              carouselData[currentSlide].title}
-          </h1>
+          <h1 className="carousel-top">{carouselData[currentSlide].course_id+": "+carouselData[currentSlide].title}</h1>
           <ul className="carousel-top">
-            {[
-              ...new Set(
```
Comment (harshsepi0l) [inline src/components/Carousel/Carousel.jsx:78]: Need to have this or the faculty names display multiple times

**Review by harshsepi0l — COMMENTED** [review]
Yeah there's a lot of stuff in here that can cause some changes to how the page is going to work. I can look it over with Luis tomorrow before the demo.

File: src/components/Carousel/Carousel.jsx [hunk 2]
```diff
@@ -1,107 +1,158 @@
 import React, { useEffect, useState } from "react";
 import axios from "axios";
 import { BsChevronLeft, BsChevronRight } from "react-icons/bs";
-import KColor from "../../KColor.png";
-import { Link } from "react-router-dom";
+import { Link } from 'react-router-dom';
 import "./Carousel.css";
-import "../../KColor.png";
 
-const Carousel = ({ onDataPassed }) => {
```
Comment (harshsepi0l) [inline src/components/Carousel/Carousel.jsx:9]: I need this const to be in there so the data from here gets passed to display.

**Comment by harshsepi0l** [comment]
Finished merging a different pull request in place of this. so closing it now.

**Pull request closed without merging**